importer/Uploader.py

```diff
@@ -296,7 +296,7 @@ def __init__(self, monument_object, log=None, tablename=None, live=False):
         if log is not None:
             self.log = log
         self.data = monument_object.wd_item
-        site = pywikibot.Site("wikidata", "wikidata")
+        site = site
```

Comment:
You don't need this line (which doesn't do anything). what you want to do is set `self.repo = site`.

Right now you allow `site=None` if you do that then you need a fallback (so `self.repo = site or utils.create_site_instance("wikidata", "wikidata")`).
A better solution would be to make site an eariler parameter (just before log) and require it to have a value. I would also rename it "repo" to make it clear that it needs to be a data repository.
Maybe also add a docstring explaining the parameters.